PinpointKit/PinpointKit/Sources/Screenshotter.swift

```diff
@@ -12,14 +12,14 @@ import Foundation
 public class Screenshotter {
 
     /**
-     Takes and returns a screenshot of all of an `application`’s windows displayed on a given screen.
+     Takes and returns a screenshot of all of an application’s windows displayed on a given screen.
      
      - parameter screen:      The screen to determine the screenshot size.
      - parameter application: The application to screenshot.
      
      - returns: A screenshot as a `UIImage`.
      */
-    public static func takeScreenshot(_ screen: UIScreen = UIScreen.main(), application: UIApplication = UIApplication.shared()) -> UIImage {
+    public static func takeScreenshot(of screen: UIScreen = UIScreen.main(), in application: UIApplication = UIApplication.shared()) -> UIImage {
```

Comment:
Another way to think about this would be `takeScreenshot(of application: UIApplication on screen: UIScreen)`. Think that’s any clearer?